copy.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 pbcopy=pbcopy
-command -v $pbcopy &>- || pbcopy='xclip -selection clipboard'
+command -v $pbcopy &>/dev/null || pbcopy='xclip -selection clipboard'
```

Comment:
Fix a bug here. I wanted to shutdown both stdout and stderr here. But unlike 1>&-,  &>- will create a file named '-'.  LOL